Cam movement scrip

john646 opened the pull request:
If a cube is in the scene, I apologize it can be deleted.

john646 committed:
Camera movement script

This is the code for the free camera. Attach it to the main camera and fill in all of the needed publics. DM me on discord if you need me to change anything

john646 committed:
Added Camera Movement Script

If there is a cube in the scene it can be deleted, was for testing purposes.

fastFox2 reviewed (changes requested):
Please stick to the c# [naming conventions](https://www.c-sharpcorner.com/UploadFile/8a67c0/C-Sharp-coding-standards-and-naming-conventions/)!

john646 committed:
Update cammove.cs

Updates the names

fastFox2 committed:
Script now follows our conventions.

john646 committed:
Updated camera scripts

This contains the updated scripts to meet the naming requirements, add a zoom feature and add the ability to rotate the camera.

john646 committed:
Merge branch 'main' of https://github.com/john646/Unititled-Tycoon-Game into main

john646 committed:
Further updates to camera rotation

Camera now rotates around the center instead of the outside.

john646 committed:
Delete cammove.cs

Deleted old no longer used scripts

john646 committed:
Update cammovement.cs

john646 committed:
Merge branch 'main' of https://github.com/john646/Unititled-Tycoon-Game into main
